thirdparty/jquery-ondemand/jquery.ondemand.js

```diff
@@ -6,13 +6,11 @@
  * Adds capabilities for custom X-Include-CSS and X-Include-JS HTTP headers
  * to request loading of externals alongside an ajax response.
  *
- * Requires jQuery 1.5 ($.Deferred support)
- *
- * CAUTION: Relies on customization of the 'beforeSend' callback in jQuery.ajaxSetup()
```

Comment:
This doesn't require jQuery 1.5. It's been running on 1.7 for who knows how long, but I've now upgraded it to work with 3.6 - `$.deferred` is [still very much supported](https://api.jquery.com/jquery.deferred/)

The note about `beforeSend` is unnecessary and frankly misleading here, since that customisation is included in this file anyway.